app/guides/layout.tsx

```diff
@@ -0,0 +1,24 @@
+"use server";
+
+import { getGuidePages } from "@/lib/server/cms-data";
+import { getSections } from "@/lib/nav-menu-builder";
+import NavMenu from "@/components/NavMenu";
+
+export default async function GuidesLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const guidePages = await getGuidePages();
+
+  return (
+    <>
```

Comment:
you don't need the wrapping `<></>` here since this whole component lives in the `<div className="flex flex-col items-center">`

Reply:
I had done this but didn't remove it because when I was testing, I had more on this page. I forgot to remove it. Will do!